--- hunspell.cxx.orig	2026-09-28 16:38:12.265714272 +0200
+++ hunspell-1.7.4/src/tools/hunspell.cxx	2026-09-28 16:38:12.280968289 +0200
@@ -114,6 +114,7 @@
 #include <sys/stat.h>
 #include <dirent.h>
 #include <unistd.h>
+#include <stdint.h>
 #include "../parsers/textparser.hxx"
 #include "../parsers/htmlparser.hxx"
 #include "../parsers/latexparser.hxx"
@@ -122,6 +123,12 @@
 #include "../parsers/xmlparser.hxx"
 #include "../parsers/odfparser.hxx"
 
+#if UINTPTR_MAX <= 0xffffffff
+#define USRLIB "/usr/lib"
+#else
+#define USRLIB "/usr/lib64"
+#endif
+
 #define LIBDIR                \
   DATADIR "/hunspell:"        \
   DATADIR "/myspell:"         \
@@ -129,6 +136,7 @@
   "/usr/share/hunspell:"      \
   "/usr/share/myspell:"       \
   "/usr/share/myspell/dicts:" \
+  "/usr/share/dict/ooo:"      \
   "/Library/Spelling"
 #define USEROOODIR {                       \
   ".openoffice.org/3/user/wordbook",       \
@@ -137,6 +145,7 @@
   ".config/libreoffice/4/user/wordbook",   \
   "Library/Spelling" }
 #define OOODIR                                       \
+  USRLIB "/libreoffice/share/wordbook:"              \
   "/opt/openoffice.org/basis3.0/share/dict/ooo:"     \
   "/usr/lib/openoffice.org/basis3.0/share/dict/ooo:" \
   "/opt/openoffice.org2.4/share/dict/ooo:"           \
