diff -up hunspell-1.7.2/src/tools/hunspell.cxx.1~ hunspell-1.7.2/src/tools/hunspell.cxx
--- hunspell-1.7.2/src/tools/hunspell.cxx.1~	2022-12-29 21:10:49.000000000 +0100
+++ hunspell-1.7.2/src/tools/hunspell.cxx	2026-01-04 18:52:36.643237309 +0100
@@ -107,6 +107,7 @@
 #include <sys/stat.h>
 #include <dirent.h>
 #include <unistd.h>
+#include <stdint.h>
 #include "../parsers/textparser.hxx"
 #include "../parsers/htmlparser.hxx"
 #include "../parsers/latexparser.hxx"
@@ -115,10 +116,17 @@
 #include "../parsers/xmlparser.hxx"
 #include "../parsers/odfparser.hxx"
 
+#if UINTPTR_MAX <= 0xffffffff
+#define USRLIB "/usr/lib"
+#else
+#define USRLIB "/usr/lib64"
+#endif
+
 #define LIBDIR                \
   "/usr/share/hunspell:"      \
   "/usr/share/myspell:"       \
   "/usr/share/myspell/dicts:" \
+  "/usr/share/dict/ooo:" \
   "/Library/Spelling"
 #define USEROOODIR {                  \
   ".openoffice.org/3/user/wordbook", \
@@ -126,6 +134,7 @@
   ".openoffice.org2.0/user/wordbook",\
   "Library/Spelling" }
 #define OOODIR                                       \
+  USRLIB "/libreoffice/share/wordbook:"              \
   "/opt/openoffice.org/basis3.0/share/dict/ooo:"     \
   "/usr/lib/openoffice.org/basis3.0/share/dict/ooo:" \
   "/opt/openoffice.org2.4/share/dict/ooo:"           \
