diff -Naur hunspell-1.4.1/src/tools/hunspell.cxx hunspell-1.4.1.tpg/src/tools/hunspell.cxx
--- hunspell-1.4.1/src/tools/hunspell.cxx	2016-05-03 07:55:52.000000000 +0000
+++ hunspell-1.4.1.tpg/src/tools/hunspell.cxx	2016-05-16 20:11:03.265171565 +0000
@@ -116,6 +116,7 @@
   "/usr/share/hunspell:"      \
   "/usr/share/myspell:"       \
   "/usr/share/myspell/dicts:" \
+  "/usr/share/dict/ooo:" \
   "/Library/Spelling"
 #define USEROOODIR                    \
   ".openoffice.org/3/user/wordbook:"  \
